Integrations/Interfaces/IMarketInstrument.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Interfaces
+{
+    /// <summary>
+    /// Represents market instrument 
+    /// </summary>
+    public interface IMarketInstrument
+    {
+        /// <summary>
+        /// Figi e.g.
```

Comment:
Idk whether this comment makes it obvious for everyone, but if so, why not

Reply:
I think Id means id, and Figi is example name id. This is special therm